test/cql-pytest/README.md

```diff
@@ -38,7 +39,14 @@ to control which tests to run:
   For `pytest` to support the `--count` option, you need to install a
   pytest extension: `pip install pytest-repeat`
 
-Additional useful pytest options, especially useful for debugging tests:
+#### How to run from PyCharm IDE
+* Create pytest config
+* Under `traget` select `Module name`:
+  * filename - to run all tests in the file (`test_keyspace`) 
+  * filename.testname - to run specific test (`test_keyspace.test_concurrent_create_and_drop_keyspace`) 
+* Add Pytest additional params under `Additional Arguments`
+![/images/pycharm_pytest_config](../images/pycharm_pytest_config.png)
 
+#### Additional useful Pytest options (especially useful for debugging tests):
```

Comment:
Why does every header use progressively increasing number of hash marks (#)?

Reply:
Done